ledger-api/grpc-definitions/com/digitalasset/ledger/api/v1/event.proto

```diff
@@ -26,6 +26,8 @@ message Event {
   oneof event {
     CreatedEvent created = 1;
     ArchivedEvent archived = 3;
+    EnteredViewEvent enteredView = 4;
+    LeftViewEvent leftView = 5;
```

Comment:
Given that you clean up here, it might make sense to also add the suggested changes to distinguish between actual created events (the ones where we will see an archive if it is archived) and just disclosed create events.